downstairs/src/main.rs

```diff
@@ -182,6 +182,45 @@ enum Args {
         bind_addr: SocketAddr,
     },
     Version,
+    /*
+    Measure an isolated downstairs
+    */
+    Dynamometer {
+        #[clap(long, default_value = "512", action)]
+        block_size: u64,
+
+        #[clap(short, long, name = "DIRECTORY", action)]
+        data: PathBuf,
+
+        #[clap(long, default_value = "100", action)]
+        extent_size: u64,
+
+        #[clap(long, default_value = "15", action)]
+        extent_count: u64,
+
+        #[clap(long, action)]
+        encrypted: bool,
+
+        // Number of writes to submit at one time to region_write
+        #[clap(short, long, action, default_value = "1")]
+        num_writes: usize,
+
+        // Number of samples to exit for
+        #[clap(short, long, action, default_value = "10")]
+        samples: usize,
+
+        // Flush per iops
+        #[clap(long, action, default_value = None)]
+        flush_per_iops: Option<usize>,
+
+        // Flush per blocks written
+        #[clap(long, action, default_value = None)]
+        flush_per_blocks: Option<usize>,
+
+        // Flush per ms
+        #[clap(long, action, default_value = None)]
+        flush_per_ms: Option<usize>,
```

Comment:
It would be neat to store a `Duration` here, so that it's strongly typed by the time it's parsed.  This requires a custom value parser, see [this discussion](https://stackoverflow.com/a/72314001).

Reply:
see e9f076d